Source/missiles.cpp

```diff
@@ -1261,28 +1267,30 @@ void InitMissiles()
 
 void AddHiveExplosion(int mi, Point /*src*/, Point /*dst*/, int midir, int8_t mienemy, int id, int dam)
 {
+	auto &missile = Missiles[mi];

```

Comment:
Ordinarily I'd suggest declaring variables close to where they are being used as this matches the style guide, but in this case having it right where the function begins helps with the eventual refactoring of the `AddMissile` class of functions.

Reply:
Agreed, I prefer it for that reason as well, it helps to highlight what globals are being used by the function.